### PR TITLE
Mint some shares to initial holder on protocol deploy and optimize submit.

### PR DESCRIPTION
**Pull request by folkyatina**
before:
telegram-cloud-document-2-5427247187782280867

after:
image

It broke almost every test, working to fix it.

### CODE REVIEW

**Comment by ujenjt** [comment]
photo_2023-02-13 17 09 49

File: contracts/0.4.24/Lido.sol [hunk 1]
```diff
@@ -285,8 +293,8 @@ contract Lido is Versioned, StETHPermit, AragonApp {
      * For more details see https://github.com/lidofinance/lido-improvement-proposals/blob/develop/LIPS/lip-10.md
      */
     function finalizeUpgrade_v2(address _lidoLocator, address _eip712StETH) external {
```
Comment (TheDZhon) [inline contracts/0.4.24/Lido.sol:295]: want to preserve the same invariant as for the fresh deployment (non-zero balance for `0xdead`)

File: contracts/0.4.24/Lido.sol [hunk 2]
```diff
@@ -252,8 +252,16 @@ contract Lido is Versioned, StETHPermit, AragonApp {
...
-        public onlyInit
+        public
+        payable
+        onlyInit
     {
+        (address holder, uint256 amount) = _bootstrapInitialHolder();
+        BUFFERED_ETHER_POSITION.setStorageUint256(amount);
+
+        emit Submitted(holder, amount, 0);
+        _emitTransferAfterMintingShares(holder, amount);
```
Comment (TheDZhon) [inline contracts/0.4.24/Lido.sol:263]: maybe move to `StETH`

**Review by TheDZhon — COMMENTED** [review]
The idea is awesome :+1: 

Though, want to preserve the existing test coverage.

**Review by TheDZhon — APPROVED** [review]
LGTM :heart:

**Comment by TheDZhon** [comment]
Merged to #624

**Pull request merged**